src/templates/_includes/forms/editableTable.twig

```diff
@@ -5,10 +5,20 @@
 {%- set initJs = not static and (initJs ?? true) -%}
 {%- set minRows = minRows ?? null %}
 {%- set maxRows = maxRows ?? null %}
-{%- set staticRows = static or (staticRows ?? false) or (minRows == 1 and maxRows == 1 and rows|length == 1) %}
-{%- set fixedRows = not staticRows and (minRows and minRows == maxRows and minRows == rows|length) %}
+{%- set allowAdd = allowAdd ?? true %}
+{%- set allowDelete = allowDelete ?? true %}
+{%- set allowReorder = allowReorder ?? true %}
```

Comment:
These should probably be `false` by default.

Reply:
Happy to make that change, the reason they are `true` is because that replicates the current default behaviour. 

Let me know if you still want these changes, and if so I can go an update the relevant twig implementations as well.